README.md

```diff
@@ -25,29 +25,30 @@ by github or other sites via a command line flag.
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-
 ## Installation
 
-    npm install -g doctoc
+Note this package has yet to be publish but you'll be able to eventually install it with
+
+    npm install -g 2doc2toc
```

Comment:
Probably don't need it to be global right?

Reply:
We if we do eventually publish this we can change this reflect how _we_ might use it